main.go

```diff
@@ -55,6 +58,11 @@ type Exporter struct {
 	mutex   sync.RWMutex
 }
 
+// GCP service account keys contain the Project ID
+type ServiceAccountFile struct {
```

Comment:
If we have to keep this, it's probably better as a local struct to where it's used. It's not representing a service-account-file as it's only a project-id (string).

Personally I'd define it just above where we populate it (line 176 I think)

Reply:
In that case should we just change it to use the [gjson](https://github.com/tidwall/gjson) library 

so we don't need the sturct at all since in reality we are just pulling one string out of the json doc

```
package main

import (
        "fmt"
        "github.com/tidwall/gjson"
        "io/ioutil"
)

func main() {
        c, _ := ioutil.ReadFile("/dev/shm/gcp.json")

        value := gjson.Get(string(c), "project_id")
        if value.String() == "" {
                fmt.Printf("Project is empty")
        }
        println(value.String())
}
```

Reply:
I updated this PR to leverage gjson.